Dockerfile

```diff
@@ -42,45 +43,25 @@ FROM alpine:3.16
 
 WORKDIR /app
 
-# Install bash, rethinkdb & python driver
+# Install bash, postgresql-client
 RUN apk add \
   --update \
   --no-cache \
-  --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main \
-    expat \
-    git
-
-# Install bash, rethinkdb & python driver
-RUN apk add \
-  --update \
-  --no-cache \
-    'apache2-utils>=2.4.52-r0' \
```

Comment:
`htpasswd` from `apache2-utils` is used in [placeos/local](https://github.com/PlaceOS/local) when running `./placeos start` for the first time

Reply:
https://github.com/PlaceOS/init/blob/f121c9077e4e98368316b633a6601d223d52d7ce/scripts/generate-secrets#L106